lib/DataGridRowView.php

```diff
@@ -22,41 +22,37 @@ class DataGridRowView implements DataGridRowViewInterface
     /**
      * @var CellViewInterface[]
      */
-    protected $cellViews = [];
+    private $cellViews = [];
 
     /**
      * @var mixed
      */
-    protected $source;
+    private $source;
 
     /**
-     * @var int
+     * @var int|string
      */
-    protected $index;
+    private $index;
 
     /**
-     * @param DataGridViewInterface $dataGridView
-     * @param ColumnTypeInterface[] $columns
+     * @param ColumnInterface[] $columns
+     * @param int|string $index
```

Comment:
Do we allow non-numerical indices or is this only mixed type for cases like `'0'`?

Reply:
`$index` can be any string like UUID or anything else